### PR TITLE
[ticket/16552] Roll back to FontAwesome 4.7

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-16552

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

https://tracker.phpbb.com/browse/PHPBB3-16552

### CODE REVIEW

**Comment by 3D-I** [comment]
!Set WIP

**Comment by 3D-I** [comment]
!unset WIP

**Comment by 3D-I** [comment]
I think I am done here, not tested.

File: phpBB/adm/style/admin.css [hunk 1]
```diff
@@ -2704,14 +2703,11 @@ fieldset.permissions .padding {
 }
 
 .send-stats-settings input[type=checkbox] + label:before {
-	font-family: "Font Awesome 5 Free";
```
Comment (marc1706) [inline phpBB/adm/style/admin.css:2707]: Missing `content: "\f096";`

**Comment by marc1706** [comment]
Rebased onto prep-release-3.3.1

**Comment by 3D-I** [comment]
> 
> 
> Rebased onto prep-release-3.3.1

Thanks, I was not sure so I went ahead and made the PR in any case.

**Pull request merged**